packages/proxy/lib/http/request-middleware.ts

```diff
@@ -365,8 +365,10 @@ const StripUnsupportedAcceptEncoding: RequestMiddleware = function () {
     if (doesAcceptHeadingIncludeGzip) {
       this.req.headers['accept-encoding'] = 'gzip'
     } else {
-      delete this.req.headers['accept-encoding']
+      this.req.headers['accept-encoding'] = 'identity'
     }
+  } else {
+    this.req.headers['accept-encoding'] = 'identity'
```

Comment:
If there is no accept-encoding header, would it make sense to set this to `gzip`?

Reply:
I've gone back and forth on it. The wording in the spec is so annoying because it says that a blank accept encoding "may" indicate that we accept everything, but it probably varies from use case to use case.

Reply:
Where is that at? I'm seeing this:

```
If no Accept-Encoding header field is in the request, any content coding is considered acceptable by the user agent.
```

https://www.rfc-editor.org/rfc/rfc9110#section-12.5.3-10.1

Reply:
Yeah you're right. I was looking at an old one first and then linked to the newer one :) I'll go with the newer one's verbage which I think means we should limit it to gzip.